linera-views/src/hashable_wrapper.rs

```diff
@@ -40,7 +40,7 @@ impl<C, W, O> View<C> for WrappedHashableContainerView<C, W, O>
 where
     C: Context + Send + Sync,
     ViewError: From<C::Error>,
-    W: HashableView<C> + Send,
+    W: HashableView<C> + Send + Sync,
```

Comment:
Is this intended?

Reply:
Yes.
That is required because of the use of `self.hash.lock().await`.